_overviews/scala3-book/taste-collections.md

```diff
@@ -60,15 +60,13 @@ nums.map(_.toUpperCase)               // List("ONE", "TWO")
 nums.flatMap(_.toUpperCase)           // List('O', 'N', 'E', 'T', 'W', 'O')
 ```
 
-These examples show how the “fold” and “reduce” methods are used to sum the values in a sequence of integers:
+These examples show how the “foldLeft” and “reduceLeft” methods are used to sum the values in a sequence of integers:
 
 ```scala
 val firstTen = (1 to 10).toList            // List(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
 
-firstTen.reduce(_ + _)                     // 55
 firstTen.reduceLeft(_ + _)                 // 55
-firstTen.fold(100)(_ + _)                  // 155 (100 is a “seed” value)
-firstTen.foldLeft(100)(_ + _)              // 155
+firstTen.foldLeft(100)(_ + _)              // 155 (100 is a “seed” value)
```

Comment:
I don’t think we should mention `reduce` and `fold`, which are leftovers of Scala 2.12 time.

Reply:
Thanks! I didn’t know those were considered leftovers.